invenio_search/ext.py

```diff
@@ -82,6 +79,14 @@ def templates(self):
 
         return templates
 
+    def generate_suffix(self, force=False):
```

Comment:
Since we didn't provide this dynamic suffix config variable we were talking about, this can be included in the `.current_suffix` property (and in the future if someone needs something custom, they can make a PR for it)